decidim-assemblies/app/controllers/concerns/decidim/assemblies/admin/filterable.rb

```diff
@@ -23,6 +23,26 @@ def base_query
           def extra_filters
             [:parent_id_eq]
           end
+
+          def filters
+            [:private_space_eq, :published_at_null, :decidim_assemblies_type_id_eq]
+          end
+
+          def filters_with_values
+            {
+              private_space_eq: [true, false],
+              published_at_null: [true, false],
+              decidim_assemblies_type_id_eq: AssembliesType.where(organization: current_organization).pluck(:id)
+            }
+          end
+
+          def dynamically_translated_filters
+            [:decidim_assemblies_type_id_eq]
+          end
+
+          def translated_decidim_assemblies_type_id_eq(id)
+            translated_attribute(Decidim::AssembliesType.find_by(id: id).title[I18n.locale.to_s])
```

Comment:
💬 Mmm, you don't need to use the locale when you use `translated_attribute`, right? Also, you can use `find` when searching by `id`. 

```suggestion
translated_attribute(Decidim::AssembliesType.find(id).title)
```